tests/testthat/test_sparkavro.R

```diff
@@ -8,12 +8,12 @@ if(!exists("sc")){
 }
 
 test_that("read existing avro", {
-  df <- spark_read_avro(sc, "twitter", system.file("extdata", package="sparkavro"), "twitter.avro", memory = FALSE)
+  df <- spark_read_avro(sc, "twitter", system.file("extdata/twitter.avro", package="sparkavro"), memory = FALSE)
```

Comment:
I think using extdata is the convention to rede raw data within a package. Is it necessary to change it? http://r-pkgs.had.co.nz/data.html#data-extdata

Reply:
I agree, it shouldn't be done the way I did it. The problem was that the "twitter.avro" argument was outside the call to system.file causing the code to fail. I have updated the code and kept it inside the call to system.file but as a separate argument as suggested on http://r-pkgs.had.co.nz/data.html#data-extdata